db_upgrade_scripts/mosip_otp/upgrade.sh

```
@@ -32,22 +30,17 @@ if [ "$ACTION" == "upgrade" ]; then
   UPGRADE_SCRIPT_FILE="sql/${CURRENT_VERSION}_to_${UPGRADE_VERSION}_upgrade.sql"
   if [ -f "$UPGRADE_SCRIPT_FILE" ]; then
     echo "Executing upgrade script $UPGRADE_SCRIPT_FILE"
-    if [[ "$UPGRADE_VERSION" == "1.2.0.2"  &&  "$CURRENT_VERSION" == "1.2.0.1" ]]; then
-      echo "Creating dml directory."
-      mkdir -p dml
-      PGPASSWORD=$SU_USER_PWD psql -v ON_ERROR_STOP=1 --username=$SU_USER --host=$DB_SERVERIP --port=$DB_PORT --dbname=$SOURCE_DB1_NAME -a -b -f $SOURCE_DB1_SUPPORT_FILE
-    fi
-    PGPASSWORD=$SU_USER_PWD psql -v ON_ERROR_STOP=1 --username=$SU_USER --host=$DB_SERVERIP --port=$DB_PORT --dbname=$DEFAULT_DB_NAME -a -b -f $UPGRADE_SCRIPT_FILE
+    PGPASSWORD=$SU_USER_PWD psql -v ON_ERROR_STOP=1 -v dbuserpwd="'${DBUSER_PWD}'" --username=$SU_USER --host=$DB_SERVERIP --port=$DB_PORT --dbname=$DEFAULT_DB_NAME -v primary_language_code=$PRIMARY_LANGUAGE_CODE -a -b -f $UPGRADE_SCRIPT_FILE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)